packages/data-stores/src/queries/use-site-domains-query.ts

```diff
@@ -3,6 +3,7 @@ import wpcomRequest from 'wpcom-proxy-request';
 
 export interface DomainData {
 	primary_domain: boolean;
+	blog_name: string;
```

Comment:
This field can also be added to `PartialDomainData`. In the hope that one day `applyColumnSort()` will be able to just accept partial domain data and won't need all the data loaded 🤞